### PR TITLE
NTR: improve README with instructions and best practices

### PR DESCRIPTION
**Pull request by boxblinkracer**
review notes

I've added some visual concepts to the readme, like code snippets and paragraphs

also, i figured out some kind of best practice for plugin development.
used it in a plugin of mine.
could you verify it? the benefit is to avoid fixtures from being accidentally loaded when the plugin only has production dependencies :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,133 @@ Just add it to your project via composer: "**composer require basecom/sw6-fixtur
...
+```xml
+<service id="Basecom\FixturePlugin\FixtureHelper">
+</service>
+
+<service id="MyNamespace\Fixtures\Customer\CustomerFixture">
+    <argument type="service" id="Basecom\FixturePlugin\FixtureHelper"/>
+    <argument type="service" id="customer.repository"/>
+    <tag name="basecom.fixture"/>
+</service>
+```
```
[the remark this answers is not in the document]
Comment (jkniest) [reply README.md:68]: Only if they are using Auto Wiring. I think I would keep the example, but also mention that we have Auto Wiring configured.

[the remark this answers is not in the document]
Comment (boxblinkracer) [reply README.md:68]: yeah, didnt work for me today, it probably wasnt turned on.
ill update the text :)

[the remark this answers is not in the document]
Comment (boxblinkracer) [reply README.md:68]: text angepasst :)

File: README.md [hunk 2]
```diff
@@ -9,20 +9,133 @@ Just add it to your project via composer: "**composer require basecom/sw6-fixtur
...
+bin/console fixture:load:group <name>
+```
+
+
+
+
+## Best Practices
+
+### Plugin Development
+When you want to add fixtures to a plugin that you build, you might not want to deliver these fixtures in a production version of your plugin.
```
Comment (boxblinkracer) [inline README.md:102]: bitte das noch prüfen @jdambacher  @jkniest , was ihr davon haltet.
hat soweit gut ausgesehen bei mir....aber wäre toll wenn ihr das noch verifiziert, danke

File: README.md [hunk 3]
```diff
@@ -9,20 +9,133 @@ Just add it to your project via composer: "**composer require basecom/sw6-fixtur
...
+public function groups(): array
+{
+    return [];
+}
+```
+
+
+## Register Fixtures
+If you're using service autowiring, then nothing needs to be done to register your fixtures.
+If you're not using autowiring, then pelase register your fixtures with the tag `basecom.fixture`.
```
Comment (jkniest) [inline README.md:56]:
```suggestion
If you're not using autowiring, then please register your fixtures with the tag `basecom.fixture`.
```

**Review by jkniest — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jkniest) [reply README.md:102]: Für mich hört sich das gut an ✌️

**Review by jdambacher — APPROVED** [review]

**Pull request merged**